### PR TITLE
document pod overrides

### PR DESCRIPTION
**Pull request by fhennig**

### CODE REVIEW

File: docs/modules/superset/pages/getting_started/first_steps.adoc [hunk 1]
```diff
@@ -6,36 +6,37 @@ Once you have followed the steps in  xref:getting_started/installation.adoc[] fo
...
+For testing, you can set up a PostgreSQL database with these commands:
 
 [source,bash]
 include::example$getting_started/getting_started.sh[tag=add-bitnami-repo]
 
 [source,bash]
 include::example$getting_started/getting_started.sh[tag=install-bitnami-psql]
 
-WARNING: This setup is unsuitable for production use! Follow the specific production setup instructions for one of the xref:required-external-components[supported databases] to get a production-ready database.
+WARNING: This setup is not suitable for production use.
```
Comment (adwk67) [inline docs/modules/superset/pages/getting_started/first_steps.adoc:17]:
```suggestion
WARNING: This setup is not suitable for use in production.
```

File: docs/modules/superset/pages/getting_started/first_steps.adoc [hunk 2]
```diff
@@ -6,36 +6,37 @@ Once you have followed the steps in  xref:getting_started/installation.adoc[] fo
...
 
-And apply it:
+Apply the Secret:
 
 [source,bash]
 include::example$getting_started/getting_started.sh[tag=apply-superset-credentials]
 
-The `connections.secretKey` will be used for securely signing the session cookies and can be used
-for any other security related needs by extensions. It should be a long random string of bytes.
+The `connections.secretKey` will be used for securely signing the session cookies and can be used for any other security-related needs by extensions.
```
Comment (adwk67) [inline docs/modules/superset/pages/getting_started/first_steps.adoc:33]:
```suggestion
The `connections.secretKey` will be used for securely signing the session cookies and can be used by the extensions for any other security-related needs.
```

File: docs/modules/superset/pages/getting_started/first_steps.adoc [hunk 3]
```diff
@@ -67,40 +68,43 @@ include::example$getting_started/getting_started.sh[tag=wait-superset]
...
-Then it can be opened in the browser with `http://localhost:8088`.
-
-Enter the admin credentials from the Kubernetes secret:
+Then, open `http://localhost:8088` in your browser and log in with the admin credentials from the Kubernetes secret.
 
 image::getting_started/superset-login.png[Login screen of Superset]
 
-Great! Now the Superset is already ready to use, but if you also want some sample data and dashboards to explore the functionalities Superset has to offer, continue with the next step.
+Superset is now ready for use.
+If you want to load sample data and dashboards and explore its Superset's functionality, proceed to the next step.
```
Comment (adwk67) [inline docs/modules/superset/pages/getting_started/first_steps.adoc:83]:
```suggestion
If you want to load sample data and dashboards and explore Superset's functionality, proceed to the next step.
```

File: docs/modules/superset/pages/getting_started/installation.adoc [hunk 4]
```diff
@@ -1,17 +1,18 @@
 = Installation
 
-On this page you will install the Stackable Superset Operator as well as the commons, secret and listener Operator which are
-required by all Stackable Operators.
+On this page you will install the Stackable Superset Operator as well as the commons, secret and listener operator which are required by all Stackable Operators.
```
Comment (adwk67) [inline docs/modules/superset/pages/getting_started/installation.adoc:3]:
```suggestion
On this page you will install the Stackable Superset Operator as well as the commons, secret and listener operators which are required by all Stackable Operators.
```

**Review by adwk67 — CHANGES_REQUESTED** [review]
Just minor things.

**Comment by fhennig** [comment]
Thanks! All good suggestions.

**Review by adwk67 — APPROVED** [review]
LGTM!

**Pull request merged**